tailwind.config.ts

```diff
@@ -3,25 +3,34 @@
 
 import fs from "fs";
 import path from "path";
+import formsPlugin from "@tailwindcss/forms";
 import plugin from "tailwindcss/plugin";
 
-module.exports = {
+export default {
   content: [
     "./js/**/*.js",
     "./js/**/*.ts",
     "./js/**/*.tsx",
     "./lib/orbit_web.ex",
     "./lib/orbit_web/**/*.*ex",
+    "./storybook/**/*.html",
   ],
+  darkMode: ["class", '[data-mode="dark"]'],
```

Comment:
This is very different from the [custom dark mode plugin](https://github.com/mbta/glides/blob/c368cc69414920254576a55a83deff5273f6473f/assets/tailwind.config.ts#L51 
) that Glides uses, but I guess we'll reconcile those differences if/when we ever add dark mode to the app.

Reply:
Yeah, I just went with this because it's what the write-up I was looking at used and it seemed pretty standard. I figure we can / will revisit in the future depending on what we decide for the actual implementation.